lhotse/utils.py

```diff
@@ -64,6 +67,16 @@ def non_null_dict_factory(collection):
     return asdict(dclass, dict_factory=non_null_dict_factory)
 
 
+def find_files_in_directory(path: Pathlike = '.', pattern: str = '*') -> List[str]:
```

Comment:
Perhaps should mention that '*' is supposed to be a UNIX-style shell wildcard, not a python regex,
and give a few examples of values that you think might be useful?  It's not even clear to me, as UNIX
has various types of pattern matching e.g. globs.